[I 2023-07-13 11:35:50.598 ServerApp] Package jupyterlab took 0.0000s to import
[I 2023-07-13 11:35:50.618 ServerApp] Package jupyter_lsp took 0.0185s to import
[W 2023-07-13 11:35:50.618 ServerApp] A `_jupyter_server_extension_points` function was not found in jupyter_lsp. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2023-07-13 11:35:50.625 ServerApp] Package jupyter_server_terminals took 0.0072s to import
[I 2023-07-13 11:35:50.627 ServerApp] Package jupyterlab_myst took 0.0011s to import
[I 2023-07-13 11:35:50.826 ServerApp] Package jupytext took 0.1993s to import
[I 2023-07-13 11:35:50.829 ServerApp] Package nbclassic took 0.0023s to import
[W 2023-07-13 11:35:50.834 ServerApp] A `_jupyter_server_extension_points` function was not found in nbclassic. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2023-07-13 11:35:50.834 ServerApp] Package notebook_shim took 0.0000s to import
[W 2023-07-13 11:35:50.834 ServerApp] A `_jupyter_server_extension_points` function was not found in notebook_shim. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2023-07-13 11:35:50.835 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2023-07-13 11:35:50.842 ServerApp] jupyter_server_terminals | extension was successfully linked.
[W 2023-07-13 11:35:50.843 LabApp] 'token' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2023-07-13 11:35:50.844 LabApp] 'iopub_data_rate_limit' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/traitlets/traitlets.py:2548: FutureWarning: Supporting extra quotes around strings is deprecated in traitlets 5.0. You can use '2578-31446' instead of '"2578-31446"' if you require traitlets >=5.
  warn(
[W 2023-07-13 11:35:50.848 ServerApp] ServerApp.token config is deprecated in 2.0. Use IdentityProvider.token.
[W 2023-07-13 11:35:50.848 ServerApp] ServerApp.iopub_data_rate_limit config is deprecated in 2.0. Use ZMQChannelsWebsocketConnection.iopub_data_rate_limit.
[I 2023-07-13 11:35:50.848 ServerApp] jupyterlab | extension was successfully linked.
[I 2023-07-13 11:35:50.848 ServerApp] jupyterlab_myst | extension was successfully linked.
[I 2023-07-13 11:35:50.848 ServerApp] jupytext | extension was successfully linked.
[I 2023-07-13 11:35:50.856 ServerApp] nbclassic | extension was successfully linked.
[W 2023-07-13 11:35:51.178 ServerApp] jupyter_nbextensions_configurator | error adding extension (enabled: True): The module 'jupyter_nbextensions_configurator' could not be found (No module named 'jupyter_nbextensions_configurator'). Are you sure the extension is installed?
    Traceback (most recent call last):
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/extension/manager.py", line 319, in add_extension
        extpkg = ExtensionPackage(name=extension_name, enabled=enabled)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/extension/manager.py", line 183, in __init__
        self._load_metadata()
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/extension/manager.py", line 198, in _load_metadata
        raise ExtensionModuleNotFound(msg) from None
    jupyter_server.extension.utils.ExtensionModuleNotFound: The module 'jupyter_nbextensions_configurator' could not be found (No module named 'jupyter_nbextensions_configurator'). Are you sure the extension is installed?
[I 2023-07-13 11:35:51.179 ServerApp] notebook_shim | extension was successfully linked.
/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/traitlets/traitlets.py:2548: FutureWarning: Supporting extra quotes around strings is deprecated in traitlets 5.0. You can use '2578-31446' instead of '"2578-31446"' if you require traitlets >=5.
  warn(
[I 2023-07-13 11:35:51.347 ServerApp] notebook_shim | extension was successfully loaded.
[I 2023-07-13 11:35:51.350 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2023-07-13 11:35:51.353 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2023-07-13 11:35:51.362 LabApp] JupyterLab extension loaded from /Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyterlab
[I 2023-07-13 11:35:51.362 LabApp] JupyterLab application directory is /Users/tparment/miniconda3/envs/jupyterlab-tpt/share/jupyter/lab
[I 2023-07-13 11:35:51.363 LabApp] Extension Manager is 'pypi'.
[I 2023-07-13 11:35:51.367 ServerApp] jupyterlab | extension was successfully loaded.
[I 2023-07-13 11:35:51.369 ServerApp] jupyterlab_myst | extension was successfully loaded.
[W 2023-07-13 11:35:51.369 ServerApp] [Jupytext Server Extension] Async contents managers like AsyncLargeFileManager are not supported at the moment (https://github.com/mwouts/jupytext/issues/1020). We will derive a contents manager from LargeFileManager instead.
[I 2023-07-13 11:35:51.369 ServerApp] [Jupytext Server Extension] Deriving a JupytextContentsManager from LargeFileManager
[I 2023-07-13 11:35:51.372 ServerApp] jupytext | extension was successfully loaded.
[I 2023-07-13 11:35:51.379 ServerApp] nbclassic | extension was successfully loaded.
[I 2023-07-13 11:35:51.380 ServerApp] Serving notebooks from local directory: /Users/tparment/git/jupyterlab-tpt
[I 2023-07-13 11:35:51.380 ServerApp] Jupyter Server 2.7.0 is running at:
[I 2023-07-13 11:35:51.380 ServerApp] http://localhost:57344/lab?token=...
[I 2023-07-13 11:35:51.380 ServerApp]     http://127.0.0.1:57344/lab?token=...
[I 2023-07-13 11:35:51.380 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 2023-07-13 11:35:51.870 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
[W 2023-07-13 11:35:52.339 ServerApp] 404 GET /nbextensions/nbextensions_configurator/tree_tab/main.js (ce18e26756eb46b1b450e886dc500c4c@::1) 18.56ms referer=http://localhost:57344/tree/examples
[I 2023-07-13 11:35:52.350 ServerApp] Loaded Jupytext configuration file at /Users/tparment/jupytext.toml
[W 2023-07-13 11:35:57.289 ServerApp] 404 GET /nbextensions/nbextensions_configurator/config_menu/main.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.41ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:57.298 ServerApp] 404 GET /nbextensions/courselevels/index.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.98ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[I 2023-07-13 11:35:58.239 ServerApp] Kernel started: 3a054214-fc23-42eb-8996-f00ca1742c49
[W 2023-07-13 11:35:58.262 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Battention%7D%20what/%7Battention%7D%20what.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.28ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.266 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bseealso%7D%20targets/%7Bseealso%7D%20targets.js (ce18e26756eb46b1b450e886dc500c4c@::1) 3.65ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.271 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Battention%7D/%7Battention%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.57ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.274 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bnote%7D/%7Bnote%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 5.57ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.277 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Btip%7D/%7Btip%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 5.82ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.281 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bimportant%7D/%7Bimportant%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 5.99ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.283 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bhint%7D%20my%20title/%7Bhint%7D%20my%20title.js (ce18e26756eb46b1b450e886dc500c4c@::1) 5.27ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.285 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bseealso%7D/%7Bseealso%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 6.54ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.287 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Badmonition%7D%20This%20requires%20an%20ad%20hoc%20title/%7Badmonition%7D%20This%20requires%20an%20ad%20hoc%20title.js (ce18e26756eb46b1b450e886dc500c4c@::1) 5.78ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.290 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bcaution%7D/%7Bcaution%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.52ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.292 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Battention%7D%20my%20title/%7Battention%7D%20my%20title.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.49ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.295 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bwarning%7D/%7Bwarning%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.32ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.297 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Berror%7D/%7Berror%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.18ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.299 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bdanger%7D/%7Bdanger%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 5.00ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.301 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Badmonition%7D%20Click%20the%20button%20to%20reveal.js (ce18e26756eb46b1b450e886dc500c4c@::1) 5.42ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.304 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Berror%7D%20BEWARE/%7Berror%7D%20BEWARE.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.98ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.306 ServerApp] 404 GET /static/nbclassic/components/codemirror/mode/%7Bmermaid%7D/%7Bmermaid%7D.js (ce18e26756eb46b1b450e886dc500c4c@::1) 4.23ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[W 2023-07-13 11:35:58.310 ServerApp] 404 GET /nbextensions/widgets/notebook/js/extension.js (ce18e26756eb46b1b450e886dc500c4c@::1) 1.71ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
[W 2023-07-13 11:35:58.956 ServerApp] 404 GET /static/nbclassic/components/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_AMS-Regular.otf (ce18e26756eb46b1b450e886dc500c4c@::1) 1.49ms referer=http://localhost:57344/notebooks/examples/sandbox-nb.md
[I 2023-07-13 11:35:59.187 ServerApp] Connecting to kernel 3a054214-fc23-42eb-8996-f00ca1742c49.
[I 2023-07-13 11:36:50.594 ServerApp] Starting buffering for 3a054214-fc23-42eb-8996-f00ca1742c49:25e83d5272cf4b40967c431251f7e09b
[I 2023-07-13 11:36:52.674 ServerApp] Connecting to kernel 3a054214-fc23-42eb-8996-f00ca1742c49.
[I 2023-07-13 11:36:53.019 ServerApp] Starting buffering for 3a054214-fc23-42eb-8996-f00ca1742c49:897a0369-5c9e-4f3e-be8b-37d1a410bbb4
[W 2023-07-13 11:36:53.059 ServerApp] 404 GET /api/contents/w8/w8-s8-c2-game.md?content=0&1689241013038 (::1): file or directory does not exist: 'w8/w8-s8-c2-game.md'
[W 2023-07-13 11:36:53.059 ServerApp] wrote error: "file or directory does not exist: 'w8/w8-s8-c2-game.md'"
    Traceback (most recent call last):
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
                 ^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/services/contents/handlers.py", line 122, in get
        self.contents_manager.get(
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupytext/contentsmanager.py", line 199, in get
        return self.super.get(path, content, type, format)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/services/contents/filemanager.py", line 415, in get
        raise web.HTTPError(404, four_o_four)
    tornado.web.HTTPError: HTTP 404: Not Found (file or directory does not exist: 'w8/w8-s8-c2-game.md')
[W 2023-07-13 11:36:53.062 ServerApp] 404 GET /api/contents/w8/w8-s8-c2-game.md?content=0&1689241013038 (ce18e26756eb46b1b450e886dc500c4c@::1) 6.45ms referer=http://localhost:57344/lab
[I 2023-07-13 11:36:53.091 LabApp] Build is up to date
[W 2023-07-13 11:36:53.779 ServerApp] 404 GET /api/contents/w8?1689241013775 (::1): file or directory does not exist: 'w8'
[W 2023-07-13 11:36:53.779 ServerApp] wrote error: "file or directory does not exist: 'w8'"
    Traceback (most recent call last):
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
                 ^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/services/contents/handlers.py", line 122, in get
        self.contents_manager.get(
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupytext/contentsmanager.py", line 199, in get
        return self.super.get(path, content, type, format)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/services/contents/filemanager.py", line 415, in get
        raise web.HTTPError(404, four_o_four)
    tornado.web.HTTPError: HTTP 404: Not Found (file or directory does not exist: 'w8')
[W 2023-07-13 11:36:53.780 ServerApp] 404 GET /api/contents/w8?1689241013775 (ce18e26756eb46b1b450e886dc500c4c@::1) 2.02ms referer=http://localhost:57344/lab
[I 2023-07-13 11:37:05.154 ServerApp] Connecting to kernel 3a054214-fc23-42eb-8996-f00ca1742c49.
[I 2023-07-13 11:39:04.959 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:39:04.963 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:41:05.004 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:41:05.007 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:43:05.057 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:43:05.060 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:45:05.109 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:45:05.112 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:47:05.170 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:47:05.174 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:49:05.219 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:49:05.222 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:51:05.271 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:51:05.274 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:53:05.327 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:53:05.331 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:55:05.385 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:55:05.389 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:57:05.437 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:57:05.440 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 11:58:38.310 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 11:58:38.314 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 12:00:38.401 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 12:00:38.404 ServerApp] Saving sandbox-nb.md
[I 2023-07-13 12:02:38.454 ServerApp] Saving file at /examples/sandbox-nb.md
[I 2023-07-13 12:02:38.457 ServerApp] Saving sandbox-nb.md
[W 2023-07-13 12:12:51.811 ServerApp] 404 GET /api/contents/w8/w8-s8-c2-game.md?content=0&1689241013038 (::1): file or directory does not exist: 'w8/w8-s8-c2-game.md'
[W 2023-07-13 12:12:51.811 ServerApp] wrote error: "file or directory does not exist: 'w8/w8-s8-c2-game.md'"
    Traceback (most recent call last):
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
                 ^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/services/contents/handlers.py", line 122, in get
        self.contents_manager.get(
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupytext/contentsmanager.py", line 199, in get
        return self.super.get(path, content, type, format)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/services/contents/filemanager.py", line 415, in get
        raise web.HTTPError(404, four_o_four)
    tornado.web.HTTPError: HTTP 404: Not Found (file or directory does not exist: 'w8/w8-s8-c2-game.md')
[W 2023-07-13 12:12:51.812 ServerApp] 404 GET /api/contents/w8/w8-s8-c2-game.md?content=0&1689241013038 (ce18e26756eb46b1b450e886dc500c4c@::1) 2.05ms referer=None
[W 2023-07-13 12:12:51.889 ServerApp] Content security violation: {"csp-report":{"document-uri":"http://localhost:57344/api/contents/w8/w8-s8-c2-game.md?content=0&1689241013038","referrer":"","violated-directive":"style-src-attr","effective-directive":"style-src-attr","original-policy":"frame-ancestors 'self'; report-uri /api/security/csp-report; default-src 'none'","disposition":"enforce","blocked-uri":"inline","line-number":1,"source-file":"http://localhost:57344/api/contents/w8/w8-s8-c2-game.md","status-code":404,"script-sample":""}}
[W 2023-07-13 12:12:58.855 ServerApp] 404 GET /api/contents/w8?1689241013775 (::1): file or directory does not exist: 'w8'
[W 2023-07-13 12:12:58.855 ServerApp] wrote error: "file or directory does not exist: 'w8'"
    Traceback (most recent call last):
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/tornado/web.py", line 1786, in _execute
        result = await result
                 ^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/services/contents/handlers.py", line 122, in get
        self.contents_manager.get(
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupytext/contentsmanager.py", line 199, in get
        return self.super.get(path, content, type, format)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/tparment/miniconda3/envs/jupyterlab-tpt/lib/python3.11/site-packages/jupyter_server/services/contents/filemanager.py", line 415, in get
        raise web.HTTPError(404, four_o_four)
    tornado.web.HTTPError: HTTP 404: Not Found (file or directory does not exist: 'w8')
[W 2023-07-13 12:12:58.856 ServerApp] 404 GET /api/contents/w8?1689241013775 (ce18e26756eb46b1b450e886dc500c4c@::1) 2.07ms referer=None
[W 2023-07-13 12:12:58.925 ServerApp] Content security violation: {"csp-report":{"document-uri":"http://localhost:57344/api/contents/w8?1689241013775","referrer":"","violated-directive":"style-src-attr","effective-directive":"style-src-attr","original-policy":"frame-ancestors 'self'; report-uri /api/security/csp-report; default-src 'none'","disposition":"enforce","blocked-uri":"inline","line-number":1,"source-file":"http://localhost:57344/api/contents/w8","status-code":404,"script-sample":""}}
[I 2023-07-13 12:13:14.241 ServerApp] Starting buffering for 3a054214-fc23-42eb-8996-f00ca1742c49:1e70e58f-7a34-41da-aaab-32de5b879ae8
[I 2023-07-13 13:06:16.219 ServerApp] AsyncIOLoopKernelRestarter: restarting kernel (1/5), keep random ports
[I 2023-07-13 13:06:19.241 ServerApp] AsyncIOLoopKernelRestarter: restarting kernel (2/5), keep random ports
[C 2023-07-13 13:06:21.012 ServerApp] received signal 15, stopping
[I 2023-07-13 13:06:21.013 ServerApp] Shutting down 7 extensions
[I 2023-07-13 13:06:21.014 ServerApp] Shutting down 1 kernel
[I 2023-07-13 13:06:21.015 ServerApp] Kernel shutdown: 3a054214-fc23-42eb-8996-f00ca1742c49

  _   _          _      _
 | | | |_ __  __| |__ _| |_ ___
 | |_| | '_ \/ _` / _` |  _/ -_)
  \___/| .__/\__,_\__,_|\__\___|
       |_|
                                                                           
Read the migration plan to Notebook 7 to learn about the new features and the actions to take if you are using extensions.

https://jupyter-notebook.readthedocs.io/en/latest/migrate_to_notebook7.html

Please note that updating to Notebook 7 might break some of your extensions.

